manifests/release/olm/operatorgroup.yaml.in

```diff
@@ -0,0 +1,5 @@
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: {{.Namespace}}
+  namespace: {{.Namespace}}
```

Comment:
I'm curious, if you don't specify:

```
spec:
    targetNamespaces:
        - {{.Namespace}}
```
Does this make it so the KubeVirt operator has to be watching all namespaces?